network-api/networkapi/wagtailpages/tests/buyersguide/test_article_page.py

```diff
@@ -71,15 +71,6 @@ def test_template(self):
             template_name='pages/base.html',
         )
 
-    def test_buyersguide_home_page_in_context(self):
```

Comment:
Since we are no longer passing "home_page" into context, this test is not needed.